apps/docs/content/guides/platform/manage-your-usage/egress.mdx

```diff
@@ -0,0 +1,114 @@
+---
+id: 'manage-usage-egress'
+title: 'Manage Egress usage'
+---
+
+## What you are charged for
+You are charged for the network data transmitted out of the system to a connected client. Egress is incurred by all services - Database, Authentication, Storage, Edge Functions, Realtime, and Log Drains.
+
+## How charges are calculated
+Egress is charged by byte. Charges apply only for usage exceeding the subscription plan's free quota. This quota can be used across all services (Database, Authentication, Storage etc.), also referred to as Unified Egress.
```

Comment:
I think what we also need to explain here are the different types of Egress: Database (i.e. Pooler), Storage, Realtime, Auth, API (PostgREST) and give some examples of what contributes to each type